Sources/StreamChat/ChatClient.swift

```diff
@@ -355,8 +361,7 @@ public class ChatClient {
         authenticationRepository.logOutUser()
 
         // Stop tracking active components
-        activeChannelControllers.removeAllObjects()
-        activeChannelListControllers.removeAllObjects()
+        syncRepository.resetTracking()
```

Comment:
I needed the similar tracking for the new type. `SyncRepository` also calls these remove calls within `resetTracking`.